src/lang-cql.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright (C) 2008 Google Inc.
```

Comment:
Please put yourself down as the copyright holder.  The Apache2 verbiage below then is your grant of license.